apps/frontend/src/libs/components/input/input.tsx

```diff
@@ -22,8 +27,11 @@ type Properties<T extends FieldValues> = {
 const Input = <T extends FieldValues>({
 	control,
 	errors,
+	iconName,
+	isDisplayedValue = false,
 	label,
 	name,
+	onToggle,
```

Comment:
```suggestion
	onIconClick,
```
It will not always be toggle